webpack.dev.ts

```diff
@@ -12,12 +10,7 @@ module.exports = merge(common, {
   devtool: "eval-source-map",
   devServer: {
     port: 8080,
-    static: {
-      directory: path.join(__dirname, "dist"),
-      staticOptions: {
-        redirect: false,
-      },
-    },
+    static: "./public",
```

Comment:
Måtte fotsatt ha en `devServer` her for at `index.html` og serviceworkeren skulle lastes inn riktig når man går til `localhost:8080/sykefravaer`. Chatgpt hjelp meg med å få det til å funke :)
Kan fjerne alt dette når vi migrerer til `vite` :nerd_face: